### DISCUSSION TITLE
UniFi Dream Machine Pro SE firmware requirement

### DISCUSSION THREAD

**Discussion by robbaman** [fabianishere/udm-iptv#148] [General]
Just a quick question. In the Readme.md file it states the following:

    UniFi Dream Machine Pro SE: You need Early Access firmware 2.3.7+ for multicast routing support

Is it still required to install an Early Access Firmware version? The post it links to is 8 months old and there's mention of a version 2.4.10 on the Ubiquiti community so I was just wondering if the remark on the Readme.md is just old and that the latest official firmware also works nowadays, of if it's still required to install that specific version?

**Comment by fabianishere** [comment]
I believe this feature is now also available in the official firmware. Will update the README.

**Comment by Bolten88** [comment]
I bought a brand new UDM SE last month and it was not necessary to have early access firmware. I have it running on 7.1.66 now.

**Reply by robbaman** [reply]
Thanks @Bolten88 I received mine last week and still need to configure it. Good to hear I won't be needing an EA firmware!
